src/components/field-label/field-label.visualroute.js

```diff
@@ -58,5 +58,8 @@ export const component = () => (
         horizontalConstraint="m"
       />
     </Spec>
+    <Spec label="with inverted tone">
+      <FieldLabel title="Hello" hasRequiredIndicator={true} tone="invereted" />
```

Comment:
```suggestion
      <FieldLabel title="Hello" hasRequiredIndicator={true} tone="inverted" />
```

Reply:
Thanks. I feel I mistype every third word. I need some pro keyboard. Something with Cherry clicks and stuffs.